cypress/integration/granules_spec.js

```diff
@@ -0,0 +1,117 @@
+import { shouldBeRedirectedToLogin } from '../support/assertions';
+
+describe('Dashboard Granules Page', () => {
+  describe('When not logged in', () => {
+    it('should redirect to login page', () => {
+      cy.visit('/#/granules');
+      shouldBeRedirectedToLogin();
+    });
+  });
+
+  describe('When logged in', () => {
+    beforeEach(() => {
+      cy.login();
+      cy.task('resetState');
+    });
+
+    after(() => {
+      cy.task('resetState');
+    });
+
+    it('should display a link to view granules', () => {
+      cy.visit('/');
+
+      cy.contains('nav li a', 'Granules').as('granules');
+      cy.get('@granules').should('have.attr', 'href', '#/granules');
+      cy.get('@granules').click();
+
+      cy.url().should('include', 'granules');
+      cy.contains('.heading--xlarge', 'Granules');
+      cy.contains('.heading--large', 'Granule Overview');
+
+      // shows a summary count of completed and failed granules
+      cy.get('.overview-num__wrapper ul li')
+        .first().contains('li', '243 Completed')
+        .next().contains('li', '32 Failed')
+        .next().contains('li', '14 Running');
+
+      // shows a list of granules
+      const granulesFile = './test/fake-api-fixtures/granules/index.json';
+      cy.readFile(granulesFile).as('granulesList');
+
+      cy.get('table tbody tr').as('list');
+      cy.get('@list').its('length').should('be.eq', 10);
+
+      // compare data in each row with the data from fixture
+      cy.get('@list').each(($el, index, $list) => {
+        cy.wrap($el).children().as('rows');
+        cy.get('@rows').its('length').should('be.eq', 8);
+
+        cy.get('@granulesList').its('results').then((granules) => {
+          const granule = granules[index];
+
+          cy.get('@rows').eq(1).invoke('text')
+            .should('be.eq', granule.status.replace(/^\w/, c => c.toUpperCase()));
+          cy.get('@rows').eq(1).children('a')
+            .should('have.attr', 'href')
+            .should('be.eq', `#/granules/${granule.status}`);
```

Comment:
Nitpick: does this work as `.and` instead of `.should`? If so, I think it might be clearer to change it to that.